js/app.js

```diff
@@ -13,6 +13,8 @@ var Tracker = function () {
 	this.photoArray = [];
 }
 
+var tracker = new Tracker();
```

Comment:
First thing I notice is that your tab spacing is still being translated as tabs at four spaces rather than two-space indents. Check your Sublime settings to fix this.